src/protocol/prf_sharding/mod.rs

```diff
@@ -742,20 +760,30 @@ pub mod tests {
         }
     }
 
-    fn test_output(
+    fn test_output<BK, TV>(
```

Comment:
This can be a new function called: `bitwise_bd_key_and_value`

The old thing can remain as it was just get renamed to: `decimal_bd_key_and_value`